src/Utilities/TypeTraits.hpp

```diff
@@ -8,6 +8,17 @@
 /// \ingroup TypeTraitsGroup
 /// C++ STL code present in C++20
 namespace cpp20 {
+/// \ingroup TypeTraitsGroup
```

Comment:
For anyone else looking at this, the C++20 standard does, in fact, classify this under type traits.